src/img/decoration/rays.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1272 72"><g fill="none" fill-rule="nonzero"><path fill="#FFF" d="M0 0l1272 72H0z" opacity=".3"/><path fill="#FFF" d="M0 24l1272 48H0z" opacity=".6"/><path fill="#F5F5F5" d="M0 48l1272 24H0z"/></g></svg>
```

Comment:
Are `rays` and `rays--small` just white diagonal images?  Hard to see how it's used just from the rich diff (or what they look like).  An image of their usage may be useful just to see them in action.

Reply:
They are just diagonal images, yes. I'll attach an image to the PR description and follow this PR up with a new section in the Fozzie docs to cover them.

Reply:
👍 Should they be part of `f-icons` potentially?  Just wondering as we've moved images out of fozzie in the past to keep them all in one place.